modules/cooldowns.lua

```diff
@@ -1426,6 +1422,40 @@ function Cooldowns:MakeGroupOptions(unit, group)
                                 end,
                                 order = 15
                             },
+                            trinketIcon = {
+                                type = "select",
+                                name = L["PvP Trinket Icon"],
+                                desc = L["Choose the PvP trinket icon style to display"],
+                                order = 16,
+                                style = "dropdown",
+                                values = function()
+                                    local data = CT:GetCooldownData(GladiusEx.IS_RETAIL and 336126 or 42292)

```

Comment:
I don’t like that it’s repeated but probably not a big deal.
Did the classic spell ID stay stable from tbcc, if you know?

Reply:
Yes, it was stable between TBCC-WoD. I think it changed in Legion with the PvP talent.